### PR TITLE
vcdat uses vcs.js with vtkweb

### PR DESCRIPTION
**Pull request by danlipsa**
Also: Use a conda environment that is already active.

### CODE REVIEW

File: frontend/src/index.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 <body>
     <div id="app"></div>
-    <script src="http://{vcs_js_server}/scripts/vcs.js" type="text/javascript"></script>
+    <script src="http://{vcs_js_server}/vcs.js" type="text/javascript"></script>
```
Comment (mattben) [inline frontend/src/index.html]: with these changes does the /scripts/autorun.sh script still work when deploying a devel server?

File: scripts/autorun.sh [hunk 2]
```diff
@@ -8,14 +8,14 @@ while [ -n $curpath ] && [ `basename $curpath` != "vcdat" ]; do
...
-if [ -n $CONDA_PREFIX ] && [ "$CONDA_PREFIX" = "$curpath/backend/venv" ]; then
+if [ -n $CONDA_PREFIX ]; then
     echo "Virtual Environment already loaded"
 else
     echo "Loading virtual environment..."
     source activate ${CONDA_ENV}
 fi
 
-vcs-server &
+vcs-server -p 8888 &
```
Comment (mattben) [inline scripts/autorun.sh]: By adding the port 8888 does the autorun.sh script need to be updated?

File: frontend/src/index.html (hunk 1 above)
Comment (danlipsa) [inline frontend/src/index.html]: Yes, it will still work with the vtkweb based vcs.js

File: scripts/autorun.sh (hunk 2 above)
Comment (danlipsa) [inline scripts/autorun.sh]: Same as previous comment, you need this for the new vtkweb based vcs.js

**Pull request merged**